azure-pipelines.yml

```diff
@@ -100,25 +100,6 @@ stages:
               echo "Coverage session ID: ${SESSION_ID}"
               VS=$(ls -d /c/Program\ Files*/Microsoft\ Visual\ Studio/*/Enterprise)
               echo "Visual Studio: ${VS}"
-              DIR="$VS/Common7/IDE/Extensions/Microsoft/CodeCoverage.Console"
-              if [[ -d $DIR ]]; then
-                # This is for MSVC 2022 (on windows-latest).
```

Comment:
This seems to be the opposite of what you wanted to do, this comment indicates this is for 2022, not 2019. (And the tests failed without actually running, so that tracks)

Reply:
Oops, sorry. Thanks @tacaswell for pushing the correct fix.